_dasldemos/fh-cbio-intro.md

```diff
@@ -0,0 +1,205 @@
+---
+title: cBioPortal - the Fred Hutch instance
+last_modified_at: 2024-09-13
+main_authors: sitapriyamoorthi, tefirman, laderast
+primary_reviewers: laderast, sonu, sckott
+---
+Learning Objectives
+-------------------
+
+After reading this article, you will learn:
+
+-   what **cBioPortal** is and how it can help with your research.
+-   what the **Fred Hutch instance of cBioPortal** is and steps to take to upload your data onto the instance.
+-   ways to format your data for upload into cBioPortal.
+
+**What is cBioportal?**
+-----------------------
+**Section in progress**
+-   It is an open-source web-based platform that allows for the visualization, exploration and analysis of cancer genomics data. When you upload data from your study, the portal dynamically creates graphs that allow you to analyze your data based on clinical characteristics.
+-   The public version of this instance can be accessed here: <https://www.cbioportal.org/> and hosts publicly available/published data.
+-   It is interactive and user-friendly
+-   It allows for the easy visualization of both genomic and clinical data
```

Comment:
I'd be more specific here, such as "cBioPortal enables visualizations that show the interaction between genomic features (such as genes) with clinical covariates. One example is stratifying survival on the presence or absence of particular genomic variants."

Reply:
I have this marked I will update it in a later pull. Thanks